server.js

```diff
@@ -23,7 +38,7 @@ app.get('/books/:id',function (req,res) {
 
 app.post('/books',function(req,res){
 var bookdata=req.body;
-var newBook = new Book(bookdata);
+var newBook=new Book(bookdata);
 newBook.save()
 .then(function(results) {
     console.log(results);
```

Comment:
you are not returning any, status code is missing? in both the `then` and `catch`